docs/publications_and_talks.md

```diff
@@ -0,0 +1,41 @@
+* **2022-10** [Split-U-Net: Preventing Data Leakage in Split Learning for Collaborative Multi-modal Brain Tumor Segmentation](https://arxiv.org/abs/2208.10553) ([DeCaF @ MICCAI 2022](https://decaf-workshop.github.io/decaf-2022/))
+* **2022-02** [Do Gradient Inversion Attacks Make Federated Learning Unsafe?](https://arxiv.org/abs/2202.06924) (Preprint)
+* **2021-10** [Multi-task Federated Learning for Heterogeneous Pancreas Segmentation](https://arxiv.org/abs/2108.08537) ([DCL @ MICCAI 2021](https://dcl-workshop.github.io/))
+* **2021-09** [Federated learning for predicting clinical outcomes in patients with COVID-19](https://www.nature.com/articles/s41591-021-01506-3) ([Nature Medicine](https://www.nature.com/nm/))
+* **2021-06** [Federated learning improves site performance in multicenter deep learning without data sharing](https://academic.oup.com/jamia/article-abstract/28/6/1259/6127556) ([JAMIA](https://academic.oup.com/jamia))
+* **2021-04** [Auto-FedAvg: Learnable Federated Averaging for Multi-Institutional Medical Image Segmentation](https://arxiv.org/abs/2104.10195) (Preprint)
+
+#### 2020
+* **2020-10** [Federated Learning for Breast Density Classification: A Real-World Implementation](https://arxiv.org/abs/2009.01871) ([DCL @ MICCAI 2020](https://dcl-workshop.github.io/dcl2020/index.html))
+* **2020-10** [Automated pancreas segmentation using multi-institutional collaborative deep learning](https://arxiv.org/abs/2009.13148) ([DCL @ MICCAI 2020](https://dcl-workshop.github.io/dcl2020/index.html))
+
+#### 2019
+* **2019-10** [Privacy-preserving Federated Brain Tumour Segmentation](https://arxiv.org/abs/1910.00962) ([MLMI @ MICCAI 2019](https://mlmi2019.web.unc.edu/))
+
+### Talks
+Recent talks covering federated learning research and NVIDIA FLARE.
+
+#### 2022
+* **2022-10** [Modern Tools for Collaborative Medical Image Analysis](https://drive.google.com/file/d/1hmlyG7g1SU8vhQ5wdTFhkqFO9Ty8BiYG/view?usp=sharing) ([Keynote - DART @ MICCAI 2022](https://sites.google.com/view/dart2022/home?authuser=0)) 
+* **2022-07** [Application of Federated Learning in Medical Image Processing](https://www.bilibili.com/video/BV1y14y147nc/?spm_id_from=333.337.search-card.all.click) ([VALSE Webinar](http://valser.org/article-572-1.html))
+* **2022-04** [Advanced Techniques for Collaborative Development of AI Models for Medical Imaging](https://rensselaer.webex.com/recordingservice/sites/rensselaer/recording/dd67440ba9f2103abaf900505681a58c/playback) ([IEEE EMBS Webinar Series](https://sites.google.com/view/ieee-biip-webinars/webinar-speakers))
+* **2022-03** [NVIDIA FLARE: Federated Learning Application Runtime Environment for Developing Robust AI Models](https://youtu.be/lLeULNI1nT8) ([SFBigAnalytics Meetup](https://www.meetup.com/sf-big-analytics/?_cookie-check=Efm7MGh7mO4YiV8A))
```

Comment:
Maybe also consider

* **2022-07** [NVIDIA FLARE Tutorial for Beginners](https://www.youtube.com/watch?v=8x7oY3xAgek&t=11s&ab_channel=NVIDIADeveloper) ([United Imaging Meetup]

Reply:
Great! Added.